src/main/java/org/folio/rest/service/report/utils/LookupHelper.java

```diff
@@ -223,7 +223,7 @@ public <T extends HasAccountInfo> Future<T> lookupRefundPayTransferFeeFineAction
     }
 
     return feeFineActionRepository.findActionsOfTypesForAccount(accountId,
-      List.of(REFUND, PAY, TRANSFER))
+        List.of(REFUND, PAY, TRANSFER))
       .map(this::sortFeeFineActionsByDate)
       .map(actions -> ctx.updateAccountContextWithActions(accountId, actions))
       .map(ctx)
```

Comment:
Indentation?

Reply:
Yes, this is a method parameter and is not related to the stream operations below